docs/docs/platforms/openshift/OPENSHIFT_FAQ.md

```diff
@@ -0,0 +1,112 @@
+WARNING:root:Permission problem writing '/opt/atlassian/confluence/confluence/WEB-INF/classes/seraph-config.xml'; skipping
+INFO:root:Generating /opt/atlassian/confluence/confluence/WEB-INF/classes/confluence-init.properties from template confluence-init.properties.j2
+WARNING:root:Permission problem writing '/opt/atlassian/confluence/confluence/WEB-INF/classes/confluence-init.properties'; skipping
+```
+
+The issue arises because the container's entrypoint script tries to generate and save `server.xml` and `seraph-config.xml` files but is unable to due to insufficient permissions. This is a normal behavior when Atlassian DC containers run as a non-root user. Note that both `server.xml` and `seraph-config.xml` are generated by Helm and mounted as read-only files within the container. If you need to change the default values for the `server.xml`, you can do it in `<product>.tomcatConfig` stanza in Helm values file.
+
+## No Pods Have Been Created
+
+This can happen when deploying Atlassian DC Helm charts to OpenShift with the default settings, and the service account has not been granted any SCC other than restricted.
+You will typically find the following error in Kubernetes events:
+
+```bash
+Error creating: pods "jira-0" is forbidden:
+unable to validate against any security context constraint:
+[provider "anyuid": Forbidden: not usable by user or serviceaccount,
+provider restricted-v2: .spec.securityContext.fsGroup: 
+Invalid value: []int64{2001}: 2001 is not an allowed group, provider "restricted":
+Forbidden: not usable by user or serviceaccount, provider "nonroot-v2":
+Forbidden: not usable by user or serviceaccount, provider "nonroot":
+Forbidden: not usable by user or serviceaccount, provider "hostmount-anyuid":
+Forbidden: not usable by user or serviceaccount, provider "machine-api-termination-handler":
+Forbidden: not usable by user or serviceaccount, provider "hostnetwork-v2":
+Forbidden: not usable by user or serviceaccount, provider "hostnetwork":
+Forbidden: not usable by user or serviceaccount, provider "hostaccess":
+Forbidden: not usable by user or serviceaccount, provider "node-exporter":
+Forbidden: not usable by user or serviceaccount, provider "efs-csi-scc":
+Forbidden: not usable by user or serviceaccount, provider "privileged":
+Forbidden: not usable by user or serviceaccount]
+```
+
+To fix this error, set `openshift.runWithRestrictedSCC` to true in Helm values.
+
+Another possible error in the events:
+
+```bash
+Error: container has runAsNonRoot and
+image will run as root (pod: "jira-0_atlassian", container: container)
+```
+
+This can happen if the ServiceAccount has been granted `nonroot` SCC privileges, but `runAsUser` has not been set in `securityContext`. To fix the error, use the following Helm values:
+
+```yaml
+jira:
+  securityContext:
+    fsGroup: 2001
+    runAsUser: 2001
+```
+
+See complete Helm values for the [Nonroot SCC](../OPENSHIFT/#__tabbed_1_2).
```

Comment:
I can't get to that link but I'm guessing it contains a full list of Helm values for the Nonroot security context. I'd change this to: "See the complete list of Helm values for the..."